docs/reference/repositories-stats/apis/get-repositories-stats.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[get-repositories-stats-api]]
+=== Repositories statistics API
```

Comment:
I would not call this repository stats API,  but repository metering API instead. Same goes for the naming throughout the PR as well as the PR title.